guildwars2/commerce.py

```diff
@@ -142,6 +142,72 @@ class CommerceMixin:
         except discord.Forbidden:
             await ctx.send("Issue embedding data into discord")
 
+    @tp.command(name="delivery")
+    @commands.cooldown(1, 10, BucketType.user)
+    async def tp_delivery(self, ctx):
+        """Show items and coins delivered to 
+            tradingpost for you
+            
+        Required permissions: tradingpost
+        """
+        user = ctx.author
+        #language = self.getlanguage(ctx)
+        item_id = ""
+        counter = 0
+        endpoint = "commerce/delivery/"
+
+        # Call API
+        try:
+            doc = await self.fetch_key(user, ["tradingpost"])
+            results = await self.call_api(endpoint, key=doc["key"])
+        except APIError as e:
+            return await self.error_handler(ctx, e)
+
+        # General output info
+        data = discord.Embed(description='Current deliveries', colour=self.embed_color)
+        data.set_author(name='Delivery overview of {0}'.format(doc["account_name"]))
+        data.set_thumbnail(
+            url="https://wiki.guildwars2.com/images/thumb/d/df/Black-Lion-Logo.png/300px-Black-Lion-Logo.png")
+        data.set_footer(text="Black Lion Trading Company")
+
+        coins = results["coins"]
+        items = results["items"]
+        items = items[:20]  # Get only first 20 entries
+        item_quantity = []
+        itemlist = []
+
+        # Get coins
+        if coins is 0:
+            gold = "No monnies for you"
```

Comment:
probably should change that

Reply:
Changed to "Currently no coins for pickup"